components/Event/EventNavBar.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="RightBar-container">
```

Comment:
classes name is always in snake-case. And we can rename variable to `event-navbar`